homework8.6/homework8.6/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace homework8._6
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите число a ");
+            var a = int.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите число b ");
+            var b = int.Parse(Console.ReadLine());
+
+            int s = 0;
+
+            if (a < b)
+            {
+                for (int i = a; i <= b; i++)
+                {
+                    for (int j = 1; j <= i; j++)
+                    {
+                        if (i % j == 0)
+                        {
+                            s = s + j;
+                        }
+                    }
+                }
```

Comment:
Проще писать так (если после `if` или `for` идет только один оператор, то скобки ставить не обязательно):
```suggestion
            if (a < b)
                for (int i = a; i <= b; i++)
                    for (int j = 1; j <= i; j++)
                        if (i % j == 0)
                            s += j;
```